# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/nsHttpChannel.cpp
# Commit: dc2ad0369b86
# Full Hash: dc2ad0369b8660e3a5e039e03e224122eb46dcfc
# Author: Randell Jesup <rjesup@mozilla.com>
# Date: 2024-04-09 21:19:46
# Regressor Bug: 1871963
# File Overlap Count: 1
# Description:
#   Bug 1871963: Implement zstd content-encoding support r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D205109
# ==============================================================================

diff -r 9fb8ccb2f305 -r dc2ad0369b86 netwerk/protocol/http/nsHttpChannel.cpp
--- a/netwerk/protocol/http/nsHttpChannel.cpp	Tue Apr 09 12:31:55 2024 +0000
+++ b/netwerk/protocol/http/nsHttpChannel.cpp	Tue Apr 09 12:31:56 2024 +0000
@@ -7690,6 +7690,7 @@
       return "proxy"_ns;
     }
     if (contentType.EqualsLiteral(APPLICATION_BROTLI) ||
+        contentType.EqualsLiteral(APPLICATION_ZSTD) ||
         contentType.Find("zip") != kNotFound ||
         contentType.Find("compress") != kNotFound) {
       return "compressed"_ns;